web/assets/css/blog_post.css

```diff
@@ -21,14 +21,23 @@
 
 .blog-post-author {
     margin-top: 25px;
+}
+
+.blog-post-body:before {
```

Comment:
It's because `<div class="row blog-post-author">` is wider than other elements like `.blog-post-body`, `.blog-post-leave-comments` and the line separator has to be moved from `.blog-post-author` to `.blog-post-author`. The second screenshot shows the issue - length of the separator line.

![screen shot 2016-12-15 at 16 42 11](https://cloud.githubusercontent.com/assets/3495796/21230725/32072e66-c2e6-11e6-82c0-f032484e7526.png)

![screen shot 2016-12-15 at 16 43 14](https://cloud.githubusercontent.com/assets/3495796/21230733/3733c476-c2e6-11e6-9834-11da35c6e0e5.png)